packages/web5/lib/src/vc/vc.dart

```diff
@@ -0,0 +1,153 @@
+import 'package:uuid/uuid.dart';
+import 'package:web5/web5.dart';
+
+class CredentialSchema {
+  String id;
+  String? type;
+
+  CredentialSchema({
+    required this.id,
+    this.type,
+  });
+
+  toJson() {
+    return {
+      'type': type,
+      'id': id,
+    };
+  }
+}
+
+class VerifiableCredential {
+  // https://www.w3.org/TR/vc-data-model/#contexts
+  static final baseContext = 'https://www.w3.org/2018/credentials/v1';
+  // https://www.w3.org/TR/vc-data-model/#dfn-type
+  static final baseType = 'VerifiableCredential';
+
+  // https://www.w3.org/TR/vc-data-model/#contexts
+  List<String> context;
+  // https://www.w3.org/TR/vc-data-model/#dfn-type
+  List<String> type;
+  // https://www.w3.org/TR/vc-data-model/#issuer
+  String issuer;
+  // https://www.w3.org/TR/vc-data-model/#credential-subject
+  String subject;
+  Map<String, dynamic> data;
+  // https://www.w3.org/TR/vc-data-model/#identifiers
+  String id;
+  // https://www.w3.org/TR/vc-data-model/#issuance-date
+  String issuanceDate;
+  // https://www.w3.org/TR/vc-data-model/#expiration
+  String? expirationDate;
+  // https://www.w3.org/TR/vc-data-model-2.0/#data-schemas
+  List<CredentialSchema>? credentialSchema;
+
+  VerifiableCredential({
```

Comment:
if we want people to just use `create()` and not the constructor, we can make this private 
```suggestion
  VerifiableCredential._({
```